.github/workflows/docs-language-linter.yaml

```
@@ -13,6 +13,10 @@
 # limitations under the License.
 
 name: Docs Language Linter
+
+permissions:
+  contents: read
+
```

Enables the action to post comments or annotations on pull requests
```suggestion

permissions:
  contents: read
  pull-requests: write   
```